exercises/static/exercises/follow_turtlebot/web-template/hal.py

```diff
@@ -16,7 +16,7 @@ def __init__(self):
         rospy.init_node("HAL")
     
         self.image = None
-        self.drone = DroneWrapper(name="rqt")
```

Comment:
undo all changes in this file when drone_wrapper backend is ready

Reply:
sure